OneLogin.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -41,8 +32,8 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/firebase/firebase-ios-sdk.git",
       "state" : {
-        "revision" : "5de0369ee79ad096c164eb3afeb7921d92a43b58",
-        "version" : "10.18.0"
+        "revision" : "8872dbd7d947acf757abab933da10e83c1842280",
```

Comment:
Looks like these have all been downgraded, worth upgrading locally and pushing again :)

Reply:
Oh good spot, dno why that happened, will change locally 👍